workinggroups/_posts/2015-11-04-licensing.md

```diff
@@ -3,8 +3,16 @@ title: Licensing
 layout: plain
 ---
 
-Providing recommendations and standards for software licensing has been identified as a high priority for the HSF. To this end the software licensing working group is being established.
+Since the early age of the HSF, software projects interested into participating mentioned the licensing as one important topic where the HSF could help. Many projects in our community don't have an explicit license, some have started with one license but are facing problems as the project is growing and incorporating new partners.
+
+This is recognized as a complex issue where the collaboration helps not to forget one aspect and to make decisions compatible with our international projects, with many different countries with different laws and policies are involved. Despite the HSF is only about open-source software, there are several licensing options with different advantages and drawbacks, not all of them being compatible, a problem when it comes to interoperability between packages.
+
+A working group has been formed in Spring 2015 with people from the community who had already some experiences with licensing issues. This working group, as any in the HSF, is open to new participants and contributions. All the discussions so far took place either in the HSF Startup Team (<hep-sf-startup-team@googlegroups.com>) or in the HSF Technical Forum (<hep-sf-forum@googlegroups.com>) for wider discussions.
```

Comment:
Change order - In spring 2015 a working group... can you check that the emails are done in proper Markdown?